base/abstractarraymath.jl

```diff
@@ -137,13 +137,13 @@ julia> b = [1 2; 3 4]
  1  2
  3  4
 
-julia> flipdim(b,2)
+julia> reverse(b, dims=2)
 2×2 Array{Int64,2}:
  2  1
  4  3
 ```
 """
-function flipdim(A::AbstractArray, d::Integer)
+function reverse(A::AbstractArray; dims::Integer)
```

Comment:
Doesn't this overwrite the `reverse` method for arrays with one with a required keyword argument?

Reply:
That one is only defined for `AbstractVector`

Reply:
Ah, I see, thanks.